python/cugraph/cugraph/dask/traversal/sssp.py

```diff
@@ -16,41 +16,67 @@
 from collections.abc import Iterable
 
 from dask.distributed import wait, default_client
-from cugraph.dask.common.input_utils import (get_distributed_data,
-                                             get_vertex_partition_offsets)
-from cugraph.dask.traversal import mg_sssp_wrapper as mg_sssp
+from cugraph.dask.common.input_utils import get_distributed_data
 import cugraph.dask.comms.comms as Comms
+import cupy
 import cudf
 import dask_cudf
-
-
-def call_sssp(sID,
-              data,
-              src_col_name,
-              dst_col_name,
-              num_verts,
-              num_edges,
-              vertex_partition_offsets,
-              aggregate_segment_offsets,
-              start):
-    wid = Comms.get_worker_id(sID)
-    handle = Comms.get_handle(sID)
-    local_size = len(aggregate_segment_offsets) // Comms.get_n_workers(sID)
-    segment_offsets = \
-        aggregate_segment_offsets[local_size * wid: local_size * (wid + 1)]
-    return mg_sssp.mg_sssp(data[0],
-                           src_col_name,
-                           dst_col_name,
-                           num_verts,
-                           num_edges,
-                           vertex_partition_offsets,
-                           wid,
-                           handle,
-                           segment_offsets,
-                           start)
-
-
-def sssp(input_graph, source):
+from pylibcugraph import sssp as pylibcugraph_sssp
+from pylibcugraph.experimental import (ResourceHandle,
+                                       GraphProperties,
+                                       MGGraph)
```

Comment:
We should be importing the non-experimental APIs now.

Reply:
fixed